packages/fragment-workflows/workflows-smoke-artifacts/runner-tick-storm.js

```
@@ -0,0 +1,163 @@
+  if (
+    Object.prototype.hasOwnProperty.call(value, "attempts") &&
+    Object.prototype.hasOwnProperty.call(value, "maxAttempts")
+  ) {
+    const attempts = Number(value.attempts);
+    const maxAttempts = Number(value.maxAttempts);
+    if (Number.isFinite(attempts) && Number.isFinite(maxAttempts) && attempts > maxAttempts) {
+      results.push({ attempts, maxAttempts, entry: value });
+    }
+  }
+  if (Array.isArray(value)) {
+    value.forEach((item) => collectAttemptViolations(item, results));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)